### ISSUE TITLE
Vulnerability roundup 96: nomad-0.12.3: 1 advisory [9.1]

### ISSUE THREAD

**Issue by ckauhaus** [NixOS/nixpkgs#102888]
[search](https://search.nix.gsc.io/?q=nomad&i=fosho&repos=NixOS-nixpkgs), [files](https://github.com/NixOS/nixpkgs/search?utf8=%E2%9C%93&q=nomad+in%3Apath&type=Code)

* [ ] [CVE-2020-27195](https://nvd.nist.gov/vuln/detail/CVE-2020-27195) CVSSv3=9.1 (nixos-unstable)

Scanned versions: nixos-unstable: 34ad166a830.

Cc @endocrimes
Cc @pradeepchhetri
Cc @rushmorem

**Comment by endocrimes** [comment]
Yep - Already covered by this but I'm waiting on a merge https://github.com/NixOS/nixpkgs/pull/97918
